### ISSUE TITLE
3rd party code should not exist in this repository.

### ISSUE THREAD

**Issue by derek-adair** [curiositry/mnml-ghost-theme#57]
Generally speaking it's not a huge issue to do this; I understand the appeal of not requiring a build step for users -- being able to drop this code directly into a ghost instance lowers the bar of entry significantly.  However, this generally considered bad practice for a number of reasons.

Instead, the dependencies should be linked w/ npm/yarn/whatever package management tool and the build should be served w/ a "[release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)".  Specifically, stuff like [FitVids](https://www.npmjs.com/package/fitvids) and prism.js should be removed from this repository and linked with npm.

This does add a build step, but is actually a much better experience for contributors.  If you are interested in incorporating something like this I can submit a pull request.  I plan on forking this and adding a couple bells and whistles (gulp to pre-process css/js for example).

**Comment by curiositry** [comment]
Good point @derek-adair! I tend to be a bit old-school :) A pull request would be welcome. Thanks!

**Comment by derek-adair** [comment]
I'll gladly do such a thing.
